pkg/api-server/testdata/inspect_dataplane_rules_subset.golden.json

```diff
@@ -135,11 +135,11 @@
    "origins": [
     {
      "mesh": "default",
-     "name": "mtp-1"
```

Comment:
Why has the order changed?

Reply:
That's on purpose, since I reused the new `origin(...)` method from `resourcerules.go` in `rules.go` I fixed the issue we had https://github.com/kumahq/kuma/issues/9173.